rs/execution_environment/src/query_handler.rs

```diff
@@ -282,6 +282,10 @@ impl InternalHttpQueryHandler {
     }
 }
 
+// TODO(EXC-1678): remove after release.
+/// Feature flag to enable/disable allowed viewers for canister log visibility.
+const ALLOWED_VIEWERS_ENABLED: bool = false;
```

Comment:
Not a strong opinion: I'm not sure the flag adds a lot of value. We could probably just unconditionally map `AllowedViewers` to default for now and then releasing the feature would simply be removing that code and allowing the field to go through as provided.

Reply:
It's easier for me just to flip this flag and see what tests fail and why. Because of that I put a corresponding comment to the specific test that has to be changed accordingly.